engine/admin/pruning.md

```diff
@@ -0,0 +1,187 @@
+data.
+
+```bash
+$ docker volume prune
+
+WARNING! This will remove all volumes not used by at least one container.
+Are you sure you want to continue? [y/N] y
+```
+
+By default, you are prompted to continue. To bypass the prompt, use the `-f` or
+`--force` flag.
+
+By default, all unused volumes are removed. You can limit the scope using
+the `--filter` flag. For instance, the following command only removes
+volumes older than 24 hours:
+
+```bash
+$ docker volume prune --filter "until=24h"
+```
+
+Other filtering expressions are available. See the
+[`docker volume prune` reference](/engine/reference/commandline/volume_prune.md)
+for more examples.
+
+## Prune networks
+
+Docker networks don't take up much disk space, but they do create `iptables`
+rules, bridge network devices, and routing table entries. To clean these things
+up, you can use `docker network prune` to clean up networks which aren't used
+by any containers.
+
+```bash
+$ docker network prune
+
+WARNING! This will remove all networks not used by at least one container.
+Are you sure you want to continue? [y/N] y
+```
+
+By default, you are prompted to continue. To bypass the prompt, use the `-f` or
+`--force` flag.
+
+By default, all unused networks are removed. You can limit the scope using
+the `--filter` flag. For instance, the following command only removes
+networks older than 24 hours:
+
+```bash
+$ docker network prune --filter "until=24h"
+```
+
+Other filtering expressions are available. See the
+[`docker network prune` reference](/engine/reference/commandline/network_prune.md)
+for more examples.
+
+## Prune everything
+
+The `docker system prune` command is a shortcut that prunes images, containers,
+and networks. In Docker 17.06.0 and earlier, volumes are also pruned. In Docker
+17.06.1 and higher, you must specify the `--volumes` flag for
+`docker system prune` to prune volumes.
+
+```bash
+$ docker system prune
+
+WARNING! This will remove:
+        - all stopped containers
+        - all networks not used by at least one container
+        - all dangling images
+        - all build cache
+Are you sure you want to continue? [y/N] y
+```
+
+If you are on Docker 17.06.1 or higher and want to also prune volumes, add
+the `--volumes` flag:
+
+```bash
+$ docker system prune --volumes
+
+WARNING! This will remove:
+        - all stopped containers
+        - all networks not used by at least one container
+        - all volumes not used by at least one container
+        - all dangling images
+        - all build cache
+Are you sure you want to continue? [y/N] y
+```
+
+By default, you are prompted to continue. To bypass the prompt, use the `-f` or
+`--force` flag.
+
+By default, all unused objects are removed. You can limit the scope using
+the `--filter` flag. For instance, the following command only removes
+objects older than 24 hours:
+
+```bash
+$ docker system prune --filter "until=24h"
```

Comment:
Use the `label` filter here; the "until" filter cannot be used with volumes, so will produce an error; https://github.com/docker/docker-ce/pull/154